db/seeds/db_populator.rb

```diff
@@ -249,7 +259,7 @@ def create_cases(casa_org, options)
         )
       end
 
-      random_future_court_date_count.times do |index|
+      random_zero_one_count.times do |index|
```

Comment:
I need a random zero or one count in the create_case_contacts method, so I thought it would be better to rename random_future_court_date_count to a more generic name that can be used in multiple places.